aggregator/src/aggregator/aggregate_init_tests.rs

```diff
@@ -103,73 +128,150 @@ pub(super) async fn setup_aggregate_init_test() -> AggregationJobInitTestCase {
         &test_case.task,
         &test_case.aggregation_job_init_req,
         &test_case.handler,
+        &test_case.auth_token,
     )
     .await;
     assert_eq!(response.status(), Some(Status::Ok));
 
     test_case
 }
 
-async fn setup_aggregate_init_test_without_sending_request() -> AggregationJobInitTestCase {
+pub(super) async fn setup_aggregate_init_test_without_sending_request() -> AggregationJobInitTestCase
+{
     install_test_trace_subscriber();
 
-    let task = TaskBuilder::new(QueryType::TimeInterval, VdafInstance::Fake, Role::Helper).build();
     let clock = MockClock::default();
     let ephemeral_datastore = ephemeral_datastore().await;
     let datastore = Arc::new(ephemeral_datastore.datastore(clock.clone()).await);
 
-    datastore.put_task(&task).await.unwrap();
+    let global_hpke_key = generate_test_hpke_config_and_private_key();
+    datastore
+        .run_tx(|tx| {
+            let global_hpke_key = global_hpke_key.clone();
+            Box::pin(async move {
+                tx.put_global_hpke_keypair(&global_hpke_key).await.unwrap();
+                Ok(())
+            })
+        })
+        .await
+        .unwrap();
+
+    let auth_token: AuthenticationToken = random();
+    let verify_key_init: VerifyKeyInit = random();
+
+    let tolerable_clock_skew = Duration::from_seconds(60);
+    let config = Config {
+        auth_tokens: vec![auth_token.clone()],
+        verify_key_init,
+        tolerable_clock_skew,
+        ..Default::default()
+    };
 
     let handler = aggregator_handler(
         Arc::clone(&datastore),
         clock.clone(),
         &noop_meter(),
-        Config::default(),
+        config.clone(),
     )
     .await
     .unwrap();
 
-    let aggregation_param = dummy_vdaf::AggregationParam(0);
+    let vdaf = Prio3Aes128Count::new_aes128_count(2).unwrap();
```

Comment:
I'm unhappy about using Prio3 as the test VDAF as that requires us to discard tests that exercise varying aggregation parameters (unless I'm mistaken?).

Unsure if it's really worth addressing in this PR. We can either adapt the taskprov messages to accept `dummy_vdaf` when compiled under tests, or I can try harder to use Poplar1 (the API is quite different in libprio-rs 0.10 compared to the latest).